project/common.scala

```diff
@@ -88,4 +90,16 @@ object common {
     pomIncludeRepository := { _ => false },
     publishArtifact in Test := false
   )
+
+  def customSettings = Seq(
+    // "0.8.1a" "0.7.3a"
+    scalazStreamVersion := {
+      if(sys.env.get("TRAVIS").nonEmpty){
```

Comment:
Why don't just check if `SCALAZ_STREAM_VERSION` is present? maybe it's useful for development.